src/app/components/practice-container/practice-container.component.ts

```diff
@@ -14,7 +15,8 @@ export class PracticeContainerComponent implements OnInit {
     article$: Observable<Article>;
 
     constructor(public dialog: MatDialog, public articlesService: ArticlesService) {
-        this.article$ = articlesService.getDocument();
+        // this.article$ = articlesService.getDocument();
```

Comment:
If you un-comment Line 18 and comment out Line 19, does it work for you?

Reply:
Yep I think so, do you want me to change it?

Reply:
Yes please :)